daal4py/sklearn/linear_model/_coordinate_descent_0_23.py

```diff
@@ -89,13 +68,45 @@ def _daal4py_fit_enet(self, X, y_, check_input):
     y = make2d(y_)
     _fptype = getFPType(X)
 
+    #only for dual_gap computation, it is not required for DAAL
+    self._X = X
+    self._y = y
+
     penalty_L1 = np.asarray(self.alpha*self.l1_ratio, dtype=X.dtype)
     penalty_L2 = np.asarray(self.alpha*(1.0 - self.l1_ratio), dtype=X.dtype)
     if (penalty_L1.size != 1 or penalty_L2.size != 1):
         raise ValueError("alpha or l1_ratio length is wrong")
     penalty_L1 = penalty_L1.reshape((1,-1))
     penalty_L2 = penalty_L2.reshape((1,-1))
 
+    #normalizing and centering
```

Comment:
Ok now, but this part have a lot of a time. It is necessary to understand the popularity of this case and maybe expand the daal's interfaces, it is also possible for linear/ridge regression

Reply:
yes, it is a temporary solution